### DISCUSSION TITLE
Android App PAX counter

### DISCUSSION THREAD

**Discussion by struthio** [meshtastic/Discussions#84] [Android]
Is there any way to get PAX data on Android phone ?

**Comment by rcarteraz** [comment]
> Is there any way to get PAX data on Android phone ?

I believe it shows on the node list if it's enabled for that device.

**Reply by struthio** [reply]
Have node with PAX enabled. I see additional screen on node showing BLE and WiFi device count. But on android device on Node list I see only information
`RSSI: -72 SNR: 13,8  (BAT) 98% 4,16V`
No PAX counter there.

I can go to debug panel and I see PAX messages like:
`
from: ....
to: ...
decoded {
  portnum: PAXCOUNTER_APP
  payload: "\030\241e"
  bitfield: 0
}
id: ....
rx_time: ...
rx_rssi: ...
...
`
But other than debug I cannot find such information
